components/style/properties/shorthand/inherited_text.mako.rs

```diff
@@ -21,3 +21,63 @@
         }
     }
 </%helpers:shorthand>
+
+// https://drafts.csswg.org/css-text-decor-3/#text-emphasis-property
+<%helpers:shorthand name="text-emphasis" products="gecko" sub_properties="text-emphasis-color
+    text-emphasis-style">
+    use properties::longhands::{text_emphasis_color, text_emphasis_style};
+
+    pub fn parse_value(context: &ParserContext, input: &mut Parser) -> Result<Longhands, ()> {
+        let mut color = None;
+        let mut style = None;
+
+        try!(input.try(|input| {
```

Comment:
You can just remove this, deindent the inner block and `return Err(())` below instead.